content/4_Gold/Combinatorics.problems.json

```diff
@@ -36,7 +36,22 @@
       "name": "MARBLES - Marbles",
       "url": "https://www.spoj.com/problems/MARBLES/",
       "source": "SPOJ",
-      "difficulty": "Medium",
+      "difficulty": "Easy",
```

Comment:
crazy

Reply:
is it not the most standard stars and bars problem ever :skull:

Reply:
especially when the template derangements gets "easy"

Reply:
fine, i'll allow it